contracts/token/erc721/abstract/ImmutableERC721Base.sol

```diff
@@ -206,6 +209,13 @@ abstract contract ImmutableERC721Base is
         return _totalSupply;
     }
 
+    // @dev allows owner or operator to burn a single token
+    function burn(uint256 tokenId) public override(ERC721Burnable) {
+        super.burn(tokenId);
```

Comment:
The regular ERC721 burnable doesn't use Bitmaps, how does this work here?

Reply:
The regular erc721 doesnt explicitly keep track of which token has been burnt, it just updates the ownership to the 0th address, the bitmap is just here for us to keep track if a token has been burnt or not, otherwise if only check ownership it just looks like the token hasnt been minted yet. The bitmap is just used for tracking the token status and we update it after the ownership updates in the erc721 code.